src/autocomplete/databases/trino/grammar/TrinoLexer.tokens

```diff
@@ -0,0 +1,665 @@
+ABSENT_=1
```

Comment:
yeah this is definitely a bug in generation
this file is already present in generated/ directory, it shouldn't be here

Reply:
I've removed it, thanks